t/passdata/buy_sig.finaltypetable.sexp

```diff
@@ -7,32 +7,4 @@
       (signature (type:name 'Signature))
       (tmp (type:name 'Bool))
       ((@app isValidSignature Seller digest0 signature) (type:name 'Bool))
-      ((@make-interaction
-        ((@record (participants (@list Buyer Seller)) (assets (@list DefaultToken))))
-        (digest0 price)
-        (@debug-label dlb0)
-        (deposit! Buyer (@record (DefaultToken price)))
-        (@debug-label dlb1)
-        (@ Seller (def signature (sign digest0)))
-        (publish! Seller signature)
-        (require! (@app isValidSignature Seller digest0 signature))
-        (@debug-label dlb2)
-        (withdraw! Seller (@record (DefaultToken price))))
-       (type:arrow (@list (type:name 'Participant) (type:name 'Participant) (type:name 'Digest) (type:name-subtype 'Nat (type:name 'Int))) (type:tuple (@list))))
-      ((@make-interaction
-        ((@record (participants (@list Buyer Seller)) (assets (@list DefaultToken))))
-        ((digest0 : Digest) (price : Nat))
-        ()
-        (@debug-label dlb0)
-        (deposit! Buyer (@record (DefaultToken price)))
-        (@debug-label dlb1)
-        (@ Seller (def signature () (sign digest0)))
-        (publish! Seller signature)
-        (require! (@app isValidSignature Seller digest0 signature))
-        (@debug-label dlb2)
-        (withdraw! Seller (@record (DefaultToken price))))
-       (type:arrow (@list (type:name 'Participant) (type:name 'Participant) (type:name 'Digest) (type:name-subtype 'Nat (type:name 'Int))) (type:tuple (@list))))
-      ((deposit! Buyer (@record (DefaultToken price))) (type:tuple (@list)))
-      ((require! (@app isValidSignature Seller digest0 signature)) (type:tuple (@list)))
-      ((sign digest0) (type:name 'Signature))
-      ((withdraw! Seller (@record (DefaultToken price))) (type:tuple (@list))))
+      ((sign digest0) (type:name 'Signature)))
```

Comment:
Observation: `(sign digest0)` is the expression that generates `signature`, which is also in the table, separately. On the Haskell side, it's probably safe to just drop anything whose key isn't a symbol; I suspect those will all be cases like this where we have a variable for it anyway.